gamebook/important_strings.py

```diff
@@ -21,9 +21,22 @@
 # a message that tells the player he entered a invalid user_input
 invalid_user_input_message = "this is an invalid choice. please try again"
 
+# a message that tells the player he entered an invalid trait name
+invalid_trait_name = "couldn't find the specified trait name to use: \
+{trait_name}"
+
+# the value at a next_scene field isn't a NextScene instance
+invalid_next_scene_value = "next scene field in scene must be\
+NextScene object or the next scene name"
+
 # a message requesting the player for user_input, goes before the input
 input_prompt = "your choice: "
 
 # a string used in input.ask_for_user_inputs to present the user_inputs
-user_inputs_request_format = """to {option_title} enter one of the
- following {option_user_inputs}\n"""
+user_inputs_request_format = "to {option_title} enter one of the \
+following {option_user_inputs}\n"
+
+# trait names
```

Comment:
Why are you doing this with different names than the var names?

Reply:
do you mean why `var1` instead of `power_trait`? i wasn't paying attention... 😞

Reply:
Happens to the best of us :)